### ISSUE TITLE
Add a quick intro on popup

### ISSUE THREAD

**Issue by anikarni** [izens-net/picket#11]
For first installers, we should have a quick description of how to use ("Add link to policy here!") as well as a link to our website.

**Comment by ProjectOak** [comment]
@anikarni and @jkarni, I'll take care of this one. I guess we would just have a permanent instruction there, right. Instead of only on the first session, right?

**Comment by anikarni** [comment]
I think we can go with a permanent one for now. Always something that can be improved later, but I don't see it as a priority at all. Maybe the only thing that would be nice to change once the user adds a policy is the name of the campaign they are participating in, but that can be anywhere on the popup, even keeping instructions as well. What do you think?

**Comment by ProjectOak** [comment]
Right, I agree, so what we can do is have a flow where users: add policy link > see success message > automatically return to the "add policy link" now showing at the bottom the latest campaign they've added.

Maybe what we could do in the future, post MVP, is create a "footer" for the pop-up where users see the latest policy they added and total number of policies they are subscribed to. In addition to that the footer could be clickable and link users to an overview of their policies where they can manage them (leave a campaign, have access to the comments and etc).
